yaksh/templates/yaksh/question.html

```diff
@@ -180,6 +192,25 @@
                      <input name="answer" type="radio" value="{{ test_case.options }}" />{{ test_case.options|safe }} <br/>
                      {% endfor %}
                      {% endif %}
+
+                     {% if question.type == "integer" %}
+                     Enter Integer:<br/>
+                     <input name="answer" type="textbox" id="integer" />
+                     <br/><br/>
+                     {% endif %}
+
+                     {% if question.type == "string" %}
+                     Enter Text:<br/>
+                     <textarea name="answer" id="string"></textarea>
+                     <br/><br/>
+                     {% endif %}
+
+                     {% if question.type == "float" %}
+                     Enter Decimal Value :<br/>
+                     <input name="answer" type="textbox" id="float" />
```

Comment:
Instead of using <input name="answer" type="textbox" id="float" /> we can use <input name="answer" type="number" id="float" /> for both integer and float questions. So that we dont need to check answer type after post submit.